poetry.lock

```diff
@@ -1,3 +1,20 @@
+[[package]]
```

Comment:
pls fix conflict with release branch

Reply:
@victor-nuzhniy please also update readme file with instructions how to setup changes that u applied in current PR

Reply:
Actually we've already have the instruction

Reply:
@victor-nuzhniy where? pls send the link

Reply:
Readme